srcpkgs/zr/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'zr'
+pkgname=zr
+version=1.0.2
+revision=1
+build_style=cargo
+hostmakedepends="pkgconf"
```

Comment:
i think most packages use pkg-config not pkgconf. does it specifically need pkgconf?